Sampler/Templates/Sampler/CHANGELOG.md.template

```diff
@@ -1,26 +1,30 @@
-# Changelog
+# Changelog for {ModuleName}
```

Comment:
did you mean `# Changelog for <%=$PLASTER_PARAM_ModuleName%>`?

Reply:
I just fixed that in anther file (github template), so can fix it here too. Can do in another PR.

Reply:
If I don't beat you to it ;)

Reply:
Done